src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -928,6 +938,15 @@ typedef enum {
     CONTENT_STORE_PRESSURE_POLICY_DROP_TAIL_ITEM = 1,
 } CONTENT_STORE_PRESSURE_POLICY;
 
+typedef enum {
+    //base case
+    STREAM_EVENT_NONE = 0,
+    // To trigger image generation and persistence in s3
+    STREAM_EVENT_IMAGE_GENERATION,
+    // To receive a notification that fragment has been persisted
+    STREAM_EVENT_NOTIFICATION,

```

Comment:
I wonder if instead of an enum if maybe these should be bit flags so that multiple options can be selected here in case in the future we will have more options here and if it's possible a customer may want to do multiple ones?